### ISSUE TITLE
[Sheet] Content underneath sheet is available

### ISSUE THREAD

**Issue by svinkle** [CovidShield/mobile#39]
## Issue summary

When the status Sheet is expanded, content underneath the Sheet is still available/focusable via swipe/touch navigation. Mobile screen readers announce this content as if it were a part of the visible Sheet content. This could lead to a confusing user experience.

![Screen Shot 2020-05-25 at 2 59 12 PM](https://user-images.githubusercontent.com/1392632/82837853-5797b880-9e98-11ea-9994-5eb0150d8680.png)

#### TSX

```tsx
// BottomSheet.tsx

return (
  <SheetContentsContainer
    isExpanded={isExpanded}
    toggleExpanded={toggleExpanded}
  >
    <>
      {collapsedContentWrapper}
      {expandedContentWrapper}
    </>
  </SheetContentsContainer>
);
```

## Steps to reproduce

1. Load the app into the iOS simulator
2. Expand the Sheet content
3. Enable [VoiceOver screen reader](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) on macOS
4. Use the Virtual Cursor to navigate to the [collapsed] Sheet control

### Expected

- Content underneath to **not** be available via touch/swipe.

### Actual

- Content underneath is available.

## Recommendation

A recommendation is not certain at this point. Some experimentation has been conducted on various content containers using the [`accessibilityViewIsModal` prop](https://reactnative.dev/docs/accessibility#accessibilityviewismodal-ios) without success.

Basically, if the `isExpanded` prop is `true`, content other than the Sheet should **not** be available on touch/swipe.

## Specifications

- **Component:** CovidShield [Sheet]
- **WCAG Principle:** Operable
- **WCAG SC:** 2.4.3 Focus Order
- **Severity:** Medium
- **Effort:** Unknown

**Comment by AbubakarSambo** [comment]
Hi @svinkle , please let me know your thoughts on this: https://github.com/CovidShield/mobile/pull/172

**Comment by timarney** [comment]
Believe this is the same issue:
https://github.com/cds-snc/covid-shield-mobile/issues/292

**Comment by AbubakarSambo** [comment]
yes, it is i believe. I have a somewhat hacky solution that hides the background elements when they're not in view and that way the screen reader doesn't read them

**Comment by timarney** [comment]
@svinkle it would be great if we can coordinate our efforts on this.  I can make someone from my team available but looking to avoid overlap on these fixes.  I've been working with @henrytao-me for getting other bug fixes back and forth between the repos.

**Comment by AbubakarSambo** [comment]
oh, i didn't know someone was working on it already, my bad.

**Comment by timarney** [comment]
> oh, i didn't know someone was working on it already, my bad.

We're not yet :) just dropped this back to see if anyone else was so we can coordinate with our repo downstream.  Avoiding two solutions to the same issue.

**Comment by JuliannaR** [comment]
https://github.com/cds-snc/covid-shield-mobile/issues/391
